tests/contracts/persistance/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "persistance"
+version = "0.1.0"
+authors = ["CPerezz <carlos@dusk.network>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+canonical = "0.6"
+canonical_derive = "0.6"
+dusk-abi = "0.9.0-rc"
```

Comment:
Most likely dusk-abi will be published before this got release, so you should just use `0.9` here, after the publishing.